sass/styles/_ciq-perfect-scrollbar.scss

```diff
@@ -94,6 +94,17 @@
     width: 11px;
 }
 
+.cq-scroll-panel{
```

Comment:
Why do you need to change this?

Reply:
@brucebinary On the first design, when the scroll is inactive, it should hide.

Reply:
@shayanbinary perfect scrollbar already does this by default

Reply:
@brucebinary I put this code so that scroll bar just appears when user scroll, by default it gets appear when the user hovers the scroll area.